src/meta/src/model_v2/migration/src/m20230908_072257_init.rs

```diff
@@ -0,0 +1,1152 @@
+                        ColumnDef::new(Database::Name)
+                            .string()
+                            .unique_key()
+                            .not_null(),
+                    )
+                    .foreign_key(
+                        &mut ForeignKey::create()
+                            .name("FK_database_object_id")
+                            .from(Database::Table, Database::DatabaseId)
+                            .to(Object::Table, Object::Oid)
+                            .on_delete(ForeignKeyAction::Cascade)
+                            .to_owned(),
+                    )
+                    .to_owned(),
+            )
+            .await?;
+        manager
+            .create_table(
+                MigrationTable::create()
+                    .table(Schema::Table)
+                    .col(ColumnDef::new(Schema::SchemaId).integer().primary_key())
+                    .col(ColumnDef::new(Schema::Name).string().not_null())
+                    .col(ColumnDef::new(Schema::DatabaseId).integer().not_null())
+                    .foreign_key(
+                        &mut ForeignKey::create()
+                            .name("FK_schema_database_id")
+                            .from(Schema::Table, Schema::DatabaseId)
+                            .to(Database::Table, Database::DatabaseId)
+                            .to_owned(),
+                    )
+                    .foreign_key(
+                        &mut ForeignKey::create()
+                            .name("FK_schema_object_id")
+                            .from(Schema::Table, Schema::SchemaId)
+                            .to(Object::Table, Object::Oid)
+                            .on_delete(ForeignKeyAction::Cascade)
+                            .to_owned(),
+                    )
+                    .to_owned(),
+            )
+            .await?;
+        manager
+            .create_table(
+                MigrationTable::create()
+                    .table(Fragment::Table)
+                    .col(
+                        ColumnDef::new(Fragment::FragmentId)
+                            .integer()
+                            .primary_key()
+                            .auto_increment(),
+                    )
+                    .col(ColumnDef::new(Fragment::TableId).integer().not_null())
+                    .col(
+                        ColumnDef::new(Fragment::FragmentTypeMask)
+                            .integer()
+                            .not_null(),
+                    )
+                    .col(
+                        ColumnDef::new(Fragment::DistributionType)
+                            .string()
+                            .not_null(),
+                    )
+                    .col(ColumnDef::new(Fragment::StreamNode).json().not_null())
+                    .col(ColumnDef::new(Fragment::VnodeMapping).json())
+                    .col(ColumnDef::new(Fragment::StateTableIds).json())
```

Comment:
For example, defines `json` type here and derived to array in the model of table `fragment`.